### ISSUE TITLE
Both examples on main page give error: Correlation ids do not match: sent 1, recv 2

### ISSUE THREAD

**Issue by ZmeiGorynych** [aio-libs/aiokafka#91]
Hi,

when I try to run both examples from the main page of this project, I get the following error on both the producer and the consumer: 

```Task exception was never retrieved
future: <Task finished coro=<wait_for() done, defined at /home/egor/anaconda3/lib/python3.5/asyncio/tasks.py:355> exception=CorrelationIdError('Correlation ids do not match: sent 1, recv 2',)>
Traceback (most recent call last):
  File "/home/egor/anaconda3/lib/python3.5/asyncio/tasks.py", line 239, in _step
    result = coro.send(None)
  File "/home/egor/anaconda3/lib/python3.5/asyncio/tasks.py", line 392, in wait_for
    return fut.result()
  File "/home/egor/anaconda3/lib/python3.5/asyncio/futures.py", line 274, in result
    raise self._exception
kafka.errors.CorrelationIdError: CorrelationIdError: Correlation ids do not match: sent 1, recv 2


```

Any idea how to fix it? Notebook attached, am using Kafka 0.9.0.1 and Python 3.5 on Ubuntu 16.04

Thanks a lot!
E.

[aiokafka_error_demo.ipynb.zip](https://github.com/aio-libs/aiokafka/files/638520/aiokafka_error_demo.ipynb.zip)

**Comment by ZmeiGorynych** [comment]
Any ideas what might be causing this? I'm getting this error both on Ubuntu with locally running Kafka, and on Windows 10 Home with Kafka running in Docker.

In both cases kafka-python on its own works just fine.

**Comment by ZmeiGorynych** [comment]
Thanks a lot! When is that release likely to happen?

**Comment by Drizzt1991** [comment]
We planned the release on this weekend

**Comment by Drizzt1991** [comment]
I did take a look one more time, the issue is only a bad side effect of auto version guessing and your producer is actually working just fine, only printing an error log. 
```
Message produced: partition 0; offset 10
```
If the error message is annoying specifying api_version="0.9" in the [constructor](http://aiokafka.readthedocs.io/en/stable/api.html#aiokafkaconsumer-class) should help remove it.

**Comment by ZmeiGorynych** [comment]
Thanks a lot! That workaround works for me!

**Comment by Drizzt1991** [comment]
It should be fixed as of 0.2.0. Closing
